unit/unit.go

```diff
@@ -25,6 +25,16 @@ import (
 	"github.com/coreos/fleet/Godeps/_workspace/src/github.com/coreos/go-systemd/unit"
 )
 
+const (
```

Comment:
Errors should start lowercase, and I'm not sure what this slice approach buys you. Better:
```
var (
       ErrNameInfo = errors.New("unable to extract ...")
       ...
)
```
But it doesn't make sense to define the errors in this package and return them from the other one..